packages/react-native-session-replay/android/gradle.properties

```diff
@@ -0,0 +1,4 @@
+DatadogSDKReactNativeSessionReplay_kotlinVersion=1.7.21
+DatadogSDKReactNativeSessionReplay_compileSdkVersion=31
+DatadogSDKReactNativeSessionReplay_buildToolsVersion=31.0.0
+DatadogSDKReactNativeSessionReplay_targetSdkVersion=31
```

Comment:
minor question: given that the current Android API is 34, should we target it?

Reply:
I'd like to keep this in sync with the value used in the version of the RN dev dependency (currently 0.71).
So I agree it should be at least 33 which is the default version for RN 0.71 and 0.72.

Is it ok if I bump it to 34 once we migrate the RN dev dependency to 0.73?

Reply:
My question is just theoretical, let's keep it in sync with the value used by the latest RN version we are using.